torch/nn/functional.py

```diff
@@ -2270,7 +2270,7 @@ def poisson_nll_loss(input, target, log_input=True, full=False, size_average=Non
             is set to ``False``, the losses are instead summed for each minibatch. Ignored
             when reduce is ``False``. Default: ``True``
         eps (float, optional): Small value to avoid evaluation of :math:`\log(0)` when
-            :attr:`log_input`=``False``. Default: 1e-8
+            :attr:`log_input` is ``False``. Default: 1e-8
```

Comment:
Fixes weird formatting in https://pytorch.org/docs/master/generated/torch.nn.parallel.DistributedDataParallel.html#distributeddataparallel for the ** device_ids** parameter